packages/hub/db/migrations/20220921073021534_alter-scheduled-payment-fields-2.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationBuilder } from 'node-pg-migrate';
+const TABLE = 'scheduled_payments';
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.addColumns(TABLE, {
+    user_address: { type: 'text', notNull: true },
+    creation_transaction_error: { type: 'text' },
+    cancellation_transaction_error: { type: 'text' },
+  });
+
+  pgm.createIndex(TABLE, 'user_address');
+
+  pgm.alterColumn(TABLE, 'fee_fixed_usd', { type: 'numeric', notNull: true });
+  pgm.alterColumn(TABLE, 'fee_percentage', { type: 'numeric', notNull: true });
+}
+
+export async function down(pgm: MigrationBuilder): Promise<void> {
+  pgm.dropColumn(TABLE, 'user_address');
```

Comment:
is a corresponding `dropIndex` needed first?

Reply:
When you drop column in a table, all indexes and constraints which use this column will be also dropped silently